packages/teams-js/src/public/video.ts

```diff
@@ -178,7 +178,7 @@ export namespace video {
           sendMessageToParent('video.videoEffectReadiness', [true, effectId]);
         })
         .catch((reason) => {
-          const validReason = reason in EffectFailureReason ? reason: EffectFailureReason.InitializationFailure;
+          const validReason = reason in EffectFailureReason ? reason : EffectFailureReason.InitializationFailure;
```

Comment:
This is just a prettier thing that happened automatically when I built.

Reply:
Happened to me too in a recent PR. We'll race to see who checks it in first!